code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -16,6 +16,7 @@ LINEN BINS
 	pickup_sound = 'sound/items/handling/cloth_pickup.ogg'
 	slot_flags = ITEM_SLOT_NECK
 	layer = BELOW_MOB_LAYER
+	base_pixel_z = 12
```

Comment:
Should probably use the defines/macros, but broadly this feels a bit awkward on an item? Like what about placing it anywhere else like on tables or somesuch?
Doesn't it spawning on a bed also already adjust the `pixel_y` by `DEPTH_OFFSET` to fit on the bed?
https://github.com/tgstation/tgstation/blob/1202be835cf29219c67cafcd5a4c811b41729ac2/code/game/objects/structures/bedsheet_bin.dm#L47-L50

Reply:
>Should probably use the defines/macros

Realized that, added!

>but broadly this feels a bit awkward on an item? Like what about placing it anywhere else like on tables or somesuch?
Maybe? Also thank you for finding that line. Okay, let me try a potential solution without shifting bedsheets upwards, it might not be necessary

Reply:
I thiiiiiiink the code for putting bedsheets on beds and having them be offset might be this, having looked into it a bit more:
https://github.com/tgstation/tgstation/blob/6c9cfa1289499186beabdb5a1c85c67d7dc440ac/code/game/objects/structures/bedsheet_bin.dm#L41
https://github.com/tgstation/tgstation/blob/6c9cfa1289499186beabdb5a1c85c67d7dc440ac/code/datums/elements/bed_tucking.dm#L14-L20
https://github.com/tgstation/tgstation/blob/6c9cfa1289499186beabdb5a1c85c67d7dc440ac/code/datums/elements/bed_tucking.dm#L55-L58

Which I believe is also used by other items you can put on beds.

So might be as simple as adding the right define to the `y_offset` for adding the element.

Reply:
You were right, I didn't need it after all

Reply:
o7